ttkwidgets/utilities.py

```diff
@@ -9,3 +10,119 @@ def get_assets_directory():
 
 def open_icon(icon_name):
     return ImageTk.PhotoImage(Image.open(os.path.join(get_assets_directory(), icon_name)))
+
+
+def get_widget_options(widget):
+    """
+    Gets the options from a widget
+
+    :param widget: tkinter.Widget instance to get the config options from
+    :return: dict of options that you can pass on to widget.config()
+    """
+    return {key: widget.cget(key) for key in widget.keys()}
+
+
+def copy_widget(widget, new_parent, level=0):
```

Comment:
This function and its dependency should be tested separately for all these test cases:
- [ ] Move a single widget from one parent to another
- [ ] Move a widget using grid on children from one parent to another
- [ ] Move a widget using pack on children from one parent to another
- [ ] Move a widget using place on children from one parent to another
- [ ] Move a widget using bindings from one parent to another
- [ ] Move a widget using a binding on its parent from one parent to another (and properly document its peculiarities)
- [ ] Attempt moving one widget to another `Tk` instance entirely (and throw an error if this cannot be done)

Reply:
AFAIK, `tkinter.Tk` is a singleton, so that last test doesn't matter

Reply:
Actually, there may be multiple `Tk` instances. It is a bad practice and makes running a program very complicated, but that is the point: It should be documented that this is not possible and the function should check whether that is actually what is requested and then throw a `RuntimeError` (or something). Then the test will determine whether the error is correctly thrown.